shortlister/controller.py

```diff
@@ -73,7 +73,7 @@ def edit_score_confirm(self,k=None):
         """Confirm changes to score"""
         print(f"Updated score: {self.current_criterion.name} to: {self.current_criterion.scores[int(k)]} and back to (applicant details)...\n")
```

Comment:
Should I move print statements like these to view.py instead?

Reply:
Yes, good idea.